dashboard/app/controllers/projects_controller.rb

```diff
@@ -133,6 +133,16 @@ def public
     end
   end
 
+  # GET /projects/featured
+  # Access is restricted to those with project_validator permission
+  def featured
+    if current_user && current_user.project_validator?
+      render template: 'projects/featured'
+    else
+      redirect_to '/projects/public', flash: {alert: 'Only project validators can feature projects.'}
```

Comment:
Is this alert visible to users? Is there any danger making it known to users that this is a valid internal url?

Reply:
Yes, the alert would be visible to users. I don't know if there is danger in making it known that this is valid internal url mostly because I don't expect external people to have any reason to try to guess this url. I can remove if you think it could be an issue; it's only there to give internal folks a little more info if they don't have the correct permission or don't know they need one.

Reply:
yeah that's fair. leave it as is!